webapp/client/public/css/components.css

```diff
@@ -249,39 +256,47 @@
 }
 
 .details-card .card-header .details-icon {
-    width: 100%;
-    background:  url("/icons/details_arrow_down.svg") no-repeat center;
+    width: 14px;
+    min-width: 14px;
```

Comment:
S. comment below?

Reply:
See this: https://github.com/digitalservice4germany/steuerlotse/pull/184#discussion_r708034678

Reply:
I would propose to also use flex-basis instead of width here, too.